allenact/setup.py

```diff
@@ -3,20 +3,61 @@
 
 from setuptools import find_packages, setup
 
+
+def parse_req_file(fname, start=None):
```

Comment:
Would be great if there was a little context about what type of file this was meant to read. I.e. a requirements file with possible extras like:
```
pkg_1
pkg_2
pkg_3

[extras1]
pkg_4
pkg_5

[extras2]
pkg_6
pkg_7
```

Reply:
That interpretation is correct, but I'll double check we still need it (I removed a lot of the hacks I had initially planned and I might have some leftovers).

Reply:
I'll keep it as a way to identify extras_requires, even if in `allenact` they don't seem necessary if we use pip install.